projects/vdk-heartbeat/src/vdk/internal/heartbeat/job_controller.py

```diff
@@ -458,3 +459,15 @@ def run(job_input):
             os.path.join(heartbeat_job_dir, "06_override_properties.py"), "w"
         ) as pyfile:
             pyfile.write(python_script)
+
+    @staticmethod
+    def _datetime_from_iso_format(datetime_string: str):
+        try:
+            return datetime.fromisoformat(datetime_string)
+        except ValueError as e:
+            log.info(
+                "An exception occurred while converting end_time to "
```

Comment:
let's log the value as well. datetime_string

Reply:
Done, thanks